### PR TITLE
lots of new miscellaneous mappings

### PR DESCRIPTION
**Pull request by eristhea**
spent a few hours mapping everything that caught my eye. mainly new parameters!

### CODE REVIEW

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/world/ChunkSerializer.mapping [hunk 1]
```diff
@@ -1,17 +1,36 @@
...
 	FIELD f_guswlivs LOGGER Lorg/slf4j/Logger;
+	FIELD f_hdinvszf IS_LIGHT_ON_KEY Ljava/lang/String;
+	FIELD f_jtclvfyj HEIGHTMAPS_KEY Ljava/lang/String;
+	FIELD f_lgudpsyv X_POS_KEY Ljava/lang/String;
+	FIELD f_llkmsmvw SKY_LIGHT_KEY Ljava/lang/String;
+	FIELD f_torykzsj SECTIONS_KEY Ljava/lang/String;
+	FIELD f_xylwjepg Z_POS_KEY Ljava/lang/String;
 	FIELD f_yfcoouba CODEC Lcom/mojang/serialization/Codec;
+	METHOD m_akmhowly appendTickNbt (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_hhlwcnih;Lnet/minecraft/unmapped/C_lwzmmmqr$C_kflzqqfb;)V
+		ARG 1 compound
```
Comment (IotaBread) [inline mappings/net/minecraft/world/ChunkSerializer.mapping]:
```suggestion
		ARG 1 nbt
```

File: mappings/net/minecraft/world/ChunkSerializer.mapping [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
+		ARG 2 ticksToSave
 	METHOD m_bvwxfvxl writeStructures (Lnet/minecraft/unmapped/C_ywyzxrzo;Lnet/minecraft/unmapped/C_ynrszrtu;Ljava/util/Map;Ljava/util/Map;)Lnet/minecraft/unmapped/C_hhlwcnih;
+	METHOD m_drlgdfvh (Lnet/minecraft/unmapped/C_rxhyurmy;)Ljava/lang/String;
+		ARG 0 fluid
 	METHOD m_ezqkremw toNbt ([Lit/unimi/dsi/fastutil/shorts/ShortList;)Lnet/minecraft/unmapped/C_hqhtfwlr;
 		ARG 0 lists
 	METHOD m_gcmtnulo getChunkType (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_kogtzhzt$C_lcffkbeg;
 		ARG 0 nbt
+	METHOD m_mxkjjsvz getList (Lnet/minecraft/unmapped/C_hhlwcnih;Ljava/lang/String;)Lnet/minecraft/unmapped/C_hqhtfwlr;
+		ARG 0 compound
```
Comment (IotaBread) [inline mappings/net/minecraft/world/ChunkSerializer.mapping]:
```suggestion
		ARG 0 nbt
```

File: mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping [hunk 3]
```diff
@@ -7,44 +7,69 @@ CLASS net/minecraft/unmapped/C_kgfalbim net/minecraft/world/gen/chunk/ChunkGener
 		COMMENT <p>
 		COMMENT This is used by {@link FlatChunkGenerator} to overwrite biome properties like whether lakes generate,
 		COMMENT while preserving the original biome identifier.
+	FIELD f_tgxfgajp generationSettingsGetter Ljava/util/function/Function;
```
Comment (IotaBread) [inline mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping]: Could be shortened to `settingsGetter`

**Review by IotaBread — APPROVED** [review]

File: mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping (hunk 3 above)
Comment (eristhea) [reply mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping]: I like that idea; will commit it later when I get to my main computer

**Pull request merged**